src/app/pages/bajas/form-solicitud/form-solicitud.component.ts

```diff
@@ -0,0 +1,265 @@
+      const consSalida = JSON.parse(res.Salida.Detalle).consecutivo;
+      const consEntrada = JSON.parse(res.Salida.MovimientoPadreId.Detalle).consecutivo;
+      (this.formBaja.get('elementos') as FormArray).at(index).patchValue({ id: res.Id });
+      (this.formBaja.get('elementos') as FormArray).at(index).patchValue({ nombre: res.Nombre });
+      (this.formBaja.get('elementos') as FormArray).at(index).patchValue({ marca: res.Marca });
+      (this.formBaja.get('elementos') as FormArray).at(index).patchValue({ subgrupo: res.SubgrupoCatalogoId.SubgrupoId.Nombre });
+      (this.formBaja.get('elementos') as FormArray).at(index).patchValue({ tipoBien: res.SubgrupoCatalogoId.TipoBienId.Nombre });
+      (this.formBaja.get('elementos') as FormArray).at(index).patchValue({ sede: res.Ubicacion.Sede.Nombre });
+      (this.formBaja.get('elementos') as FormArray).at(index).patchValue({ dependencia: res.Ubicacion.Dependencia.Nombre });
+      (this.formBaja.get('elementos') as FormArray).at(index).patchValue({ ubicacion: res.Ubicacion.Ubicacion.EspacioFisicoId.Nombre });
+      (this.formBaja.get('elementos') as FormArray).at(index).patchValue({ funcionario: res.Funcionario.Tercero.NombreCompleto });
+      (this.formBaja.get('elementos') as FormArray).at(index).patchValue({ entrada: consSalida });
+      (this.formBaja.get('elementos') as FormArray).at(index).patchValue({ salida: consEntrada });
+    });
+  }
+
+  private submitForm(statusChanges: Observable<any>) {
+    statusChanges
+      .debounceTime(250)
+      .subscribe(() => {
+        this.valid.emit(this.formBaja.valid);
+        if (this.formBaja.valid && this.load) {
+          this.bajaInfoChange.emit(this.formBaja);
+        }
+      });
+  }
+
+  public muestraDependencia(field) {
+    return field ? field.Placa : '';
+  }
+
+  private loadPlacas(text: string) {
+    const queryOptions$ = text.length > 3 ?
+      this.bajasHelper.getElementos(text) :
+      new Observable((obs) => { obs.next([{}]); });
+    return combineLatest([queryOptions$]).pipe(
+      map(([queryOptions_$]) => ({
+        queryOptions: queryOptions_$,
+      })),
+    );
+  }
+
+  private validateObjectCompleter(): ValidatorFn {
+    return (control: AbstractControl): ValidationErrors | null => {
+      const valor = control.value;
+      const checkStringLength = typeof (valor) === 'string' && valor.length < 4 && valor !== '';
+      const checkInvalidString = typeof (valor) === 'string' && valor !== '';
+      const checkInvalidObject = typeof (valor) === 'object' && !valor.Id;
+      return checkStringLength ? { errorLongitudMinima: true } :
+        ((checkInvalidString || checkInvalidObject) ? { dependenciaNoValido: true } : null);
+    };
+  }
+
+  private validateElementos(): ValidatorFn {
+    return (control: AbstractControl): ValidationErrors | null => {
+      const elementos = control.value.length;
+      return !elementos ? { errorNoElementos: true } : null;
```

Comment:
![image](https://user-images.githubusercontent.com/6588872/144172436-3c549edf-6733-44c2-975c-c5242d44b294.png)
Es una bobada pero
```suggestion
      return elementos ? null : { errorNoElementos: true };
```